### PR TITLE
v0.9.10

### PR DESCRIPTION
**Pull request by jamieadkins95**
* Changed PATCH to "v0-9-10"
* Added check for null abilities (Some abilities are declared but not defined)
* Added new categories

### CODE REVIEW

File: master_xml.py [hunk 1]
```diff
@@ -372,7 +374,46 @@ def removeUnreleasedCards(cards):
...
+              "Relict": "Relict",
+              "Shapeshifter": "Shapeshifter",
+              "Soldier": "Soldier",
+              "Special": "Special",
+              "Spell": "Spell",
+              "Spy": "Agent",
+              "Support": "Support",
+              "Svalblod": "Svalblod",
+              "Tactic": "Tactic",
+              "Temeria": "Temeria",
```
Comment (jamieadkins95) [inline master_xml.py]: Duplicated "Temeria"

File: master_xml.py [hunk 2]
```diff
@@ -372,7 +374,46 @@ def removeUnreleasedCards(cards):
...
+              "Leader": "Leader",
+              "Medic": "Medic",
+              "Non_Medicable": "Permadeath",
+              "Officer": "Officer",
+              "Ogroid": "Ogroid",
+              "Organic": "Organic",
+              "Potion": "Potion",
+              "Redania": "Redania",
+              "Regressing": "Regressing",
+              "Relict": "Relict",
```
Comment (Zolmeister) [inline master_xml.py]: dupe

File: master_xml.py [hunk 3]
```diff
@@ -372,7 +374,46 @@ def removeUnreleasedCards(cards):
...
+              "Spell": "Spell",
+              "Spy": "Agent",
+              "Support": "Support",
+              "Svalblod": "Svalblod",
+              "Tactic": "Tactic",
+              "Temeria": "Temeria",
+              "Tordarroch": "Tordarroch",
+              "Tuirseach": "Tuirseach",
+              "Vodyanoi": "Vodyanoi",
+              "Weather": "Weather"}
```
Comment (Zolmeister) [inline master_xml.py]: dupe... just put this through a JSON validator

**Review by Zolmeister — CHANGES_REQUESTED** [review]

File: master_xml.py (hunk 3 above)
Comment (jamieadkins95) [reply master_xml.py]: Fixed. Sorry was in a rush last night when I typed these up. I've removed dupes and alphabetised them.

**Review by Zolmeister — APPROVED** [review]

**Review by Tri125 — APPROVED** [review]

**Pull request merged**